### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by cyoq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +18,10 @@ description
...
-
-# What else has to be done
\ No newline at end of file
+# What else has to be done
+Some of the features which were not implemented yet:
+- [ ] Zooming and panning for the graphs. Something similiar to [Highcharts](https://www.highcharts.com/demo/dynamic-master-detail). The current implementation with Chart.js does not allow to create easy to use zooming and panning.
+- [ ] Events  page. The idea is to allow user to choose conditions how the Smart Power Socket should work. For example, if the power is more than 500W, then the socket should stop working.
+- [ ] Better UI/UX. It is not possible to use Tabs for navigation as well as better design, as UI was created by technicians, not designers.
+- [ ] Tests are not present. Therefore, it is quite important to write some tests for all parts of the project in the future. 
+- [ ] Due to lockdown in the spring 2020, it was not possible to create real Smart Power Socket. Therefore, there is a lot of work to create our own smart device.
```
Comment (Ri0ee) [inline README.md:27]: ...real Smart Power Socket **device**.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Smart Power Socket
-description
+ - It is a small project of the future students of the university during which we tried to create our own smart device and the environment for its work. 
+ 
+# Goal of the project
+  1) Create a Smart Power Socket device which will be able to measure power input and collect statistics about power usage.
+  2) Create software ecosystem where user ir able to turn on/off device, to browse the statistics of power usage and to create conditions which the smart device should execute.
```
Comment (OlegGedzjuns) [inline README.md:6]:
```suggestion
...where user **is** able to ..
```

Comment (OlegGedzjuns) [inline README.md:6]:
```suggestion
  2) Create software ecosystem where user is able to turn on/off device, to browse the statistics of power usage and to create conditions which the smart device should execute.
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+  3) Gain new knowledge in working with microcontrollers and creating full-stack web application.
```
Comment (OlegGedzjuns) [inline README.md:7]:
```suggestion
  3) Gain new knowledge in working with microcontrollers and creating full-stack web applications.  
```

File: README.md (hunk 1 above)
Comment (OlegGedzjuns) [inline README.md:27]:
```suggestion
- [ ] Due to lockdown in the spring 2020, it was not possible to create real Smart Power Socket device. Therefore, there is a lot of work to create our own smart device. 
```

File: backend/README.md [hunk 4]
```diff
@@ -10,57 +10,105 @@
...
+    -   `SocketsController.php` - controls addition, deletion, updating of the socket itself and the information about it.
+    -   `UsersController.php` - controls deletion and updating for user data.
+
+## API requests routes
+
+They are described in `routes/api.php`.
+
+-   `auth` routes:
+    -   POST `api/auth/login` - **{ email: string, password: string }** - handles user logging into the system.
+    -   POST `api/auth/register` - **{ name: string, email: string, password }** - handles user registration into the system.
```
Comment (OlegGedzjuns) [inline backend/README.md:101]:
```suggestion
    -   POST `api/auth/register` - **{ name: string, email: string, password: string }** - handles user registration into the system.
```

File: backend/README.md [hunk 5]
```diff
@@ -10,57 +10,105 @@
...
+    -   GET `api/auth/refresh` - handles JWT refresh.
+-   `sockets` routes:
+    -   POST `api/sockets/add` - **{ unique_id: number }** - handles the addition of the new Smart Power Socket.
+    -   PUT `api/sockets/connect` - **{ name: string, description?: string, unique_id: number }** - connects new Smart Power Socket from the frontend.
+    -   GET `api/sockets/list` - shows all available for a user sockets.
```
Comment (OlegGedzjuns) [inline backend/README.md:106]:
```suggestion
    -   GET `api/sockets/list` - shows all sockets available for a user.
```

File: README.md (hunk 1 above)
Comment (OlegGedzjuns) [inline README.md:27]:
```suggestion
- [ ] Due to lockdown in the spring 2020, it was not possible to create real Smart Power Socket device. Therefore, there is a lot of work to create our own smart device. 
```

**Pull request merged**